learn/what_is_meilisearch/language.md

```diff
@@ -25,13 +25,7 @@ If you'd like to help by developing a tokenizer pipeline yourself: first of all,
 
 ### What do you mean when you say Meilisearch offers _optimized_ support for a language?
 
-Under the hood, Meilisearch relies on tokenizers that identify the most important parts of each document in a given dataset. We currently use five tokenization pipelines:
-
-- A default pipeline designed for languages that separate words with spaces
-- A pipeline specifically tailored for Chinese
-- A pipeline specifically tailored for Hebrew
-- A pipeline specifically tailored for Japanese
-- A pipeline specifically tailored for Thai
+Under the hood, Meilisearch relies on tokenizers that identify the most important parts of each document in a given dataset. Optimized support for a language means Meilisearch has implemented a tokenizer specifically tailored for that language.
```

Comment:
> that identify the most important parts of each document in a given dataset

I'm not sure to understand the meaning of this sentence. 🤔 

> Optimized support for a language means Meilisearch has implemented a tokenizer specifically tailored for that language

If you want to be more precise without exhaustively listing every language. You could say that an optimized/supported language has at least a dedicated/specialized segmentation or a dedicated/specialized normalization what do you think?

Reply:
I was working under the assumption the text before the pipeline list had already been approved. If those bits are also not accurate, I propose we rewrite this answer to remove mentions of tokenizers and pipelines, replacing them with a generic "internal processes".

What do you think about this?

```suggestion
Optimized support for a language means Meilisearch has implemented internal processes specifically tailored to parsing that language, leading to more relevant results.
```

Reply:
No problem! I'll update the PR, then 👍